app/frontend/src/views/form/Success.vue

```diff
@@ -8,7 +8,8 @@
             Your form has been submitted successfully
           </h1>
           <h3>
-            Please keep the following Confirmation ID for your records:
+            If you wish to keep a record of this submission, you can keep the
+            following Confirmation ID:
```

Comment:
Could do `Your submission's Confirmation ID is:` - if we state it factually instead of as a hypothetical, it could achieve the same thing and also be a shorter message.

Reply:
Need to drive home that they really don't need it for this form. Going to be changed in short order anyways when we remove it for the COVID form